exercises/tournament/src/main/java/Tournament.java

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
You only need this started implementation if this exercises is one of the first 20 exercises on the track (i.e. one of the first 20 exercises in `config.json`), which I don't think this should be as it's relatively difficult. See our [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations) for policies like this one. So please remove this file :) Unless you think it should be one of the first 20 exercises. When you add an entry to `config.json` for this exercise you can estimate the difficulty for it :)